msteams-platform/toolkit/Teams-Toolkit-CLI.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Teams Toolkit Command Line Interface
 author: MuyangAmigo
-description: In this module, learn Teamsapp Command Line Interface, TeamsFx library, supported commands and its scenarios.
+description: In this module, learn Teamsapp Command Line Interface, TeamsFx library, supported commands, and its scenarios.
 ms.author: zhany
```

Comment:
Can we remove TeamsFx library? This is CLI doc, should not concern TeamsFx anymore, which was our SDK.

Reply:
removed.